frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java

```diff
@@ -38,7 +37,7 @@
  *  <li>{@link CommonDataKeys#PROJECT}</li>
  * </ul>
  */
-public abstract class BaseRebaseBranchOntoParentAction extends DumbAwareAction {
+public abstract class BaseRebaseBranchOntoParentAction extends GitMacheteRepositoryReadyAction {
```

Comment:
Remove the `KEY_IS_GIT_MACHETE_REPOSITORY_READY` check from `update` in this class then